ui/src/app/components/status-alert-banner.tsx

```diff
@@ -18,17 +19,14 @@ const styles = reactStyles({
     position: 'absolute',
     top: '0',
     right: '0',
-    zIndex: 1,
+    zIndex: 101,
   }
 });
 
-
-
 export interface Props {
-  statusAlertId: number;
   title: string;
   message: string;
-  link: string;
+  footer: JSX.Element;
```

Comment:
ooh neat. Is the ability to pass elements around built into React or is this something new you brought in?

Reply:
it's built in! Another common way react elements are passed around is through `props.children` which is automatically added to every react component. `props.children` will be set to whatever is inside of a JSX tag.

Ex.
```
<Container>
  <div>
    ... w/e JSX content you want
  <div>
</Container>
```

props.children within the `Container` component will be equal to the `div` JSX element.


https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx

Reply:
I'd seen that and figured it was a special case (which, I guess, it is.)